lib/rack/attack.rb

```diff
@@ -128,6 +128,17 @@ def call(env)
         configuration.tracked?(request)
         @app.call(env)
       end
+    rescue *allowed_errors
```

Comment:
On second thought, wouldn't this rescue dalli and redis errors ocurring inside other middlewares down the line called in `@app.call(env)`?

Reply:
You're right, I've refactored the scope which needs error handling into its own method.